analyses/molecular-subtyping-EPN/README.md

```diff
@@ -11,93 +21,84 @@ This above  script is designed to change to this directory to run, so it should
 
 ## Folder content
 
-1. <b>`00-subset-for-EPN.R`</b> is a script that takes subsets only  ependymoma samples expression data for CI. The script uses `pbta-histologies.tsv` file to filter for ependymoma samples and     `pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds` file for expression  data
+1. __`00-subset-for-EPN.R`__ is a script that subsets expression data to only include ependymoma samples for CI. The script uses `pbta-histologies.tsv` file to filter for ependymoma samples and `pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds` file for expression data.
 
-2. <b>`01-make_notebook_RNAandDNA.py`</b> is a script that filters WGS  and RNA-seq ependymoma samples from `pbta-histologies.tsv` file and adds `disease_group` column to the output file based on the primary_site. The values for `disease_group` are supratentorial/infratentorial/undetermined
+2. __`01-assign-disease-group.R`__ is a script that filters WGS and RNA-seq ependymoma samples from `pbta-histologies.tsv` file and adds `disease_group` column to the output file based on the primary_site. The values for `disease_group` are: 1) supratentorial, 2) infratentorial, 3) spinal, 4) mixed, and 5) undetermined.
 
-3. <b>`02_ependymoma_generate_all_data.py`</b>  is a script that takes in expression, GISTIC, fusion, breakpoint, GISTIC, GSVA files to add values from these tables as new columns to the input notebook. Output from `01-make_notebook_RNAandDNA.py` script is used as input notebook. The output notebook from this is saved to `results/EPN_all_data.tsv`
+3. __`02_ependymoma_generate_all_data.R`__  is a script that takes in expression, GISTIC, consensus focal CNV, fusion, breakpoint, GSVA, and SNV files to add values from these tables as new columns to the input notebook. Output from `01-assign-disease-group.R` script is used as input notebook. The output notebook from this is saved to `results/EPN_all_data.tsv`
```

Comment:
`GISTIC` appears twice in this line.

Reply:
just in the old code--new code only has GISTIC once.